### PR TITLE
Add BUPT and faculty

### PR DESCRIPTION
**Pull request by oneliey**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-h.csv [hunk 1]
```diff
@@ -701,6 +704,7 @@ Hongfei Fu 0001,Shanghai Jiao Tong University,http://jhc.sjtu.edu.cn/~hongfeifu,
 Hongfei Yan,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=6010,NOSCHOLARPAGE
 Hongfeng Yu,University of Nebraska,http://vis.unl.edu/~yu,yeWsFW4AAAAJ
 Hongfu Liu,Brandeis University,http://hongfuliu.com,3FqYBw0AAAAJ
+Honggang Zhang 0002,BUPT,http://www.pris.net.cn/introduction/teacher/zhanghonggang,j55tWpwAAAAJ
```
Comment (emeryberger) [inline csrankings-h.csv:707]: Not clearly in CS, please clarify.

File: csrankings-f.csv [hunk 2]
```diff
@@ -163,6 +166,7 @@ Fei Miao,University of Connecticut,http://feimiao.org,fH2YF6YAAAAJ
 Fei Peng,Hunan University,http://csee.hnu.edu.cn/people/pengfei,FT0YdtIAAAAJ
 Fei Sha,University of Southern California,http://www-bcf.usc.edu/~feisha,8rttV7kAAAAJ
 Fei Song,University of Guelph,https://www.uoguelph.ca/computing/people/fei-song,NOSCHOLARPAGE
+Fei Su,BUPT,https://teacher.bupt.edu.cn/sufei/en/index/36998/list/index.htm,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-f.csv:169]: Not clearly in CS; please clarify.

File: csrankings-c.csv [hunk 3]
```diff
@@ -970,6 +975,7 @@ Chun Xue,City University of Hong Kong,https://www.cs.cityu.edu.hk/~jasonxue,bVQN
 Chun Yan Miao,Nanyang Technological University,http://www.ntulily.org/ascymiao,fmXGRJgAAAAJ
 Chun Yang,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=5938,NOSCHOLARPAGE
 Chun Yu,Tsinghua University,http://pi.cs.tsinghua.edu.cn/lab/people/ChunYu,NOSCHOLARPAGE
+Chun-Guang Li,BUPT,http://www.pris.net.cn/en/cgli,KhXx4joAAAAJ
```
Comment (emeryberger) [inline csrankings-c.csv:978]: Not clearly in CS; please clarify.

**Review by emeryberger — CHANGES_REQUESTED** [review]
I found many faculty who do not clearly have a School of Computer Science affiliation. Please show how everyone with a non scs address meets the inclusion criteria.

File: csrankings-h.csv (hunk 1 above)
Comment (oneliey) [reply csrankings-h.csv:707]: Honggang Zhang is in AI department, which is belong to school of CS. And he can advise PhD students in CS. He advises students to publish papers at Computer Vision conferences, such as ICCV, ECCV and CVPR. Thanks~

1. Lan Yang, Kaiyue Pang, **Honggang Zhang** , Yi-Zhe Song, SketchAA: Abstract Representation for Abstract Sketches, in Proc. of International Conference on Computer Vision (ICCV), Canada, Oct. 11-17, 2021
2. Yang Tang, Wangding Zeng, **Honggang Zhang**, PIAP-DF: Pixel-Interested and Anti Person-Specific Facial Action Unit Detection Net with Discrete Feedback Learning, in Proc. of International Conference on Computer Vision (ICCV), Canada, Oct. 11-17, 2021
3. Xiaolin Song, Kaili Zhao, Wen-sheng Chu, **Honggang Zhang**, Jun Guo, Progressive Refinement Network for Occluded Pedestrian Detection, in Proc. European Conference on Computer Vision (ECCV), Aug. 23-28, 2020
4. Pang, Ke. Li, Y. Yang, **Honggang Zhang**, T. Hospedales, T. Xiang and Y. Song, Generalising Fine-Grained Sketch-Based Image Retrieval, in Proc. of IEEE Conference on Computer Vision and Pattern Recognition(CVPR), Long Beach, USA, Jun. 16-20, 2019.

File: csrankings-f.csv (hunk 2 above)
Comment (oneliey) [reply csrankings-f.csv:169]: Fei Su is also in Artificial Intelligence department, which is belong to School of CS. She advises Phd students to publish papers at Computer Vision conference.
1. Z. Meng, Z. Zhao, **F. Su** and L. Guo, "Hierarchical Spatial Pyramid Network For Cervical Precancerous Segmentation By Reconstructing Deep Segmentation Networks," 2021 IEEE/CVF Conference on Computer Vision and Pattern Recognition
2. W. Yang, Y. Song, Z. Zhao and **F. Su**, "Instance Search via Fusing Hierarchical Multi-level Retrieval and Human-object Interaction Detection," 2021 IEEE/CVF International Conference on Computer Vision Workshops (ICCVW), 2021, pp. 2323-2327, doi: 10.1109/ICCVW54120.2021.00263.

File: csrankings-c.csv (hunk 3 above)
Comment (oneliey) [reply csrankings-c.csv:978]: Chun-Guang Li is currently an Associate Professor with the Artificial Intelligence department, which is belong to School of CS. His research interests focus on computer vision. He can advise Phd students to publish papers at Computer Vision conference.
1. Ying Chen, **Chun-Guang Li**, and Chong You, “Stochastic Sparse Subspace Clustering”, IEEE/CVF Conference on Computer Vision and Pattern Recognition (CVPR), 2020, pp.4155-4164.
2. Chong You, **Chun-Guang Li**, Daniel P. Robinson, and Rene Vidal, “Is an Affine Constraint Needed for Affine Subspace Clustering?”, In Proc. of IEEE International Conference on Computer Vision(ICCV) 2019, pp.9915-9924, Oct. 27-Nov. 2, 2019, Seoul, Korea. 
3. Junjian Zhang, **Chun-Guang Li,** Chong You, Xianbiao Qi, Honggang Zhang, Jun Guo, Zhouchen Lin, “Self-Supervised Convolutional Subspace Clustering Network”, IEEE Conference on Computer Vision and Pattern Recognition (CVPR), pages 5473-5482, Jun. 16-20, 2019, Long Beach, California, USA.

**Comment by oneliey** [comment]
I have clarified all requests. Thank you for comments and reviewing

File: csrankings-h.csv (hunk 1 above)
Comment (emeryberger) [reply csrankings-h.csv:707]: I just want to clarify for the record that publishing papers in CS conferences is not related to whether the faculty in question can be included. It is only a question of whether faculty can solely supervise PhDs in CS.

**Pull request merged**